build.gradle

```diff
@@ -567,56 +566,49 @@ jlink {
     // The module descriptor automatically generated by the plugin for the merged module contained some invalid entries.
     // Execute ./gradlew suggestMergedModuleInfo and include the suggested directives here.
     mergedModule {
-        requires 'java.xml'
-        requires 'com.sun.xml.txw2'
-        requires 'java.desktop'
-        requires 'org.objectweb.asm'
-        requires 'jdk.jfr'
-        requires 'jdk.unsupported'
-        requires 'com.google.gson'
-        requires 'java.management'
-        requires 'java.naming'
-        requires 'java.security.sasl'
-        requires 'jdk.security.jgss'
-        requires 'java.logging'
-        requires 'jdk.xml.dom'
-        requires 'java.sql'
-        requires 'java.rmi'
-        requires 'java.security.jgss'
-        requires 'com.oracle.truffle.regex'
-        requires 'jdk.jsobject'
-        requires 'jdk.management'
-        requires 'jdk.unsupported.desktop'
-        requires 'org.objectweb.asm.util'
-        requires 'java.scripting'
-        requires 'org.objectweb.asm.commons'
-        requires 'org.graalvm.truffle'
-        requires 'java.datatransfer'
-        requires 'java.compiler'
-        requires 'java.sql.rowset'
-        requires 'java.transaction.xa'
-        requires 'org.graalvm.sdk'
+        requires 'jdk.xml.dom';
+        requires 'java.sql';
+        requires 'java.rmi';
+        requires 'java.xml';
+        requires 'com.sun.xml.txw2';
+        requires 'java.desktop';
+        requires 'java.security.jgss';
+        requires 'jdk.jfr';
+        requires 'jdk.jsobject';
+        requires 'jdk.unsupported';
+        requires 'java.net.http';
+        requires 'com.google.gson';
+        requires 'java.management';
+        requires 'java.naming';
+        requires 'jdk.unsupported.desktop';
+        requires 'java.security.sasl';
+        requires 'java.scripting';
+        requires 'java.datatransfer';
+        requires 'java.compiler';
```

Comment:
Here, everything seems to be changed. Could we have some alphabetical ordering for the non-grouped dependencies?

Reply:
now sorted alphabetically